data-esnext.js

```diff
@@ -716,21 +716,23 @@ exports.tests = [
   name: 'class decorators',
   category: STAGE2,
   significance: 'medium',
-  spec: 'https://github.com/wycats/javascript-decorators',
+  spec: 'https://tc39.github.io/proposal-decorators/',
   exec: function(){/*
     class A {
       @nonconf
       get B() {}
     }
-    function nonconf(target, name, descriptor) {
-      descriptor.configurable = false;
-      return descriptor;
+    function nonconf(target) {
+      target.descriptor.configurable = false;
+      target.key = 'C';
+      return target;
     }
-    return Object.getOwnPropertyDescriptor(A.prototype, "B").configurable === false;
+    return Object.getOwnPropertyDescriptor(A.prototype, 'B') === undefined &&
+           Object.getOwnPropertyDescriptor(A.prototype, 'C').configurable === false;
   */},
   res: {
-    babel: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Babel 6 still has no official support decorators, but you can use <a href='https://github.com/loganfsmyth/regenerator-plugin-transform-decorators-legacy'>this plugin</a>."},
-    typescript: true,
+    babel: {val: false, note_id: "regenerator-decorators-legacy", note_html: "Stage 2 decorators are in progress, see <a href='https://github.com/babel/babel/issues/2645'>babel/babel#2645</a>."},
```

Comment:
People don't have to be stupid to get confused by the stage process. Not everyone who uses JavaScript is an expert in TC39's process. And this website is consumed by a lot of everyday developers.

People *are* confused about what exactly "stage-0 decorators" and "stage-2 decorators" means. I know capable engineers who wrote a bunch of code against stage-0 decorators assuming that it'll mostly continue to work against stage-2 decorators. It's not like that's an unreasonable expectations, given the precedent set by other proposals that had little to no changes while going through the stages.

The audience of the kangax table is very diverse, so I think it shouldn't just dump all the information w/o considering the impact.